gordo_components/data_provider/providers.py

```diff
@@ -23,6 +23,10 @@
 logger = logging.getLogger(__name__)
 
 
+class NoSuitableDataProviderError(ValueError):
+    pass
```

Comment:
Do you think we should have a common collection for gordo exceptions?

Reply:
By collection do you mean to put them somewhere together? IDK, I feel like errors like these belongs in only class which throws them, they are part of the API of this class. But more generall errors can definitely be collected somewhere.

Reply:
Ya, I mean libs like `sklearn` have them in `sklearn.exceptions` and in general that seems fairly standard. But I do not have strong opinions about this, yet. :-)